sharding-orchestration/sharding-orchestration-reg/sharding-orchestration-reg-nacos/src/main/java/org/apache/shardingsphere/orchestration/reg/nacos/NacosRegistryCenter.java

```diff
@@ -80,7 +81,7 @@ public boolean isExisted(final String key) {
 
     @Override
     public List<String> getChildrenKeys(final String key) {
-        return null;
+        return Collections.singletonList(getDirectly(key));

```

Comment:
I think the getChildrenKeys method should get all data with the prefix of the parameter `key`.
e.g: three keys:  
/schema/masterslave_db
/schema/masterslave_db/datasource
/schema/masterslave_db/rule
if the parameter `key` is `/schema/masterslave_db`, it should get the `datasource` and `rule` data.

Reply:
![image](https://user-images.githubusercontent.com/15086151/67552967-c106eb80-f73e-11e9-86ad-269a3e5f5474.png)
![image](https://user-images.githubusercontent.com/15086151/67553010-d3812500-f73e-11e9-8088-1396f588c43d.png)
In the Bootstrap Code , you can see that schemename is used.